src/components/MainPlot/index.tsx

```diff
@@ -151,7 +168,10 @@ export default class MainPlot extends React.Component<MainPlotProps, MainPlotSta
 
         return annotations.map((point, index) => {
             const lastOne = index + 1 === annotations.length;
-            const show = lastOne && this.state.showFullAnnotation;
+            // FIXME: There's probably a better way to prevent this.state being
+            // undefined here and crashing the app, but this is a quick fix to get
+            // the tabbed view working.
+            const show = this.state ? lastOne && this.state.showFullAnnotation : false;
```

Comment:
when is this.state defined for the first time?

Reply:
oh, I hadn't looked at this before, I'll check it out. I think it's because this function is being called in the constructor